### PR TITLE
Add SQS icons

### PR DESCRIPTION
**Pull request by ranyoo2367**
Added SQS icons for queues and tool window.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Description

- Icon for SQS queue: 16x16
- Icon for SQS tool window: 13x13

## Testing

No tests created.

## Screenshots (if appropriate)
Screen Shot 2020-07-16 at 11 30 38 AM

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] Local run of `gradlew check` succeeds
- [x] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/resources/icons/resources/sqs/SqsQueue.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="17px" viewBox="0 0 18 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Amazon simple queue service (SQS) - light</title>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Amazon-simple-queue-service-(SQS)---light" transform="translate(0.866390, 0.254418)" fill="#999999" fill-rule="nonzero">
```
Comment (hunterwerlla) [inline jetbrains-core/resources/icons/resources/sqs/SqsQueue.svg:5]: I always get this confused but this is backwards. the 999999 one should be `_dark` one.

File: jetbrains-core/resources/icons/resources/sqs/SqsToolWindow.svg [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="14px" height="13px" viewBox="0 0 18 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Amazon simple queue service (SQS) - light</title>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Amazon-simple-queue-service-(SQS)---light" transform="translate(0.866390, 0.254418)" fill="#999999" fill-rule="nonzero">
```
Comment (hunterwerlla) [inline jetbrains-core/resources/icons/resources/sqs/SqsToolWindow.svg:5]: same with them backwards

File: jetbrains-core/resources/icons/resources/sqs/SqsToolWindow_dark.svg [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="14px" height="13px" viewBox="0 0 18 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (abrooksv) [inline jetbrains-core/resources/icons/resources/sqs/SqsToolWindow_dark.svg:2]: This should be 13x13

File: jetbrains-core/resources/icons/resources/sqs/SqsQueue_dark.svg [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="17px" viewBox="0 0 18 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (abrooksv) [inline jetbrains-core/resources/icons/resources/sqs/SqsQueue_dark.svg:2]: 16x16

Comment (ranyoo2367) [reply jetbrains-core/resources/icons/resources/sqs/SqsQueue_dark.svg:2]: I see, should I manually change the width and height?

Comment (ranyoo2367) [reply jetbrains-core/resources/icons/resources/sqs/SqsQueue_dark.svg]: It is fixed.

**Pull request merged**